### DISCUSSION TITLE
redis无法ping的aws集群

### DISCUSSION THREAD

**Discussion by iyyzh** [zeromicro/go-zero#3039] [Q&A]
由于aws设计问题，是无法ping但是可以使用的，go-zero在1.5.0版本加入了ping不通就启动不了的逻辑，建议把这个东西删除或者只做提示？

**Comment by auula** [comment]
> 由于aws设计问题，是无法ping但是可以使用的，go-zero在1.5.0版本加入了ping不通就启动不了的逻辑，建议把这个东西删除或者只做提示？

我也觉得可以通过一个参数来配置启动，针对不同场景来实现不同实现，如果你觉得我答案对你有用，你可以点击对你有帮助按钮，🙏。
